### PR TITLE
Add capabilities for performance tuning

### PR DESCRIPTION
**Pull request by Savidude**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,3 +157,10 @@ Add the configurations to the `custom.yml`. A sample is given below.
 ```
 
 Follow the steps mentioned under `docs` directory to customize/create new Ansible scripts and deploy the recommended patterns.
+
+## Performance Tuning
+
+System configurations can be changed through Puppet to optimize OS level performance. Performance tuning can be enabled by changing `enable_performance_tuning` in `dev/group_vars/apim.yml` to `true`.
```
Comment (VimukthiPerera) [inline README.md]: `System configurations can be changed through Ansible`

File: dev/group_vars/apim.yml [hunk 2]
```diff
@@ -22,6 +22,14 @@ target: /mnt
 product_package_location: files
 #jdbc_driver: mysql-connector-java-5.1.47-bin.jar
 
+# Performance tuning configurations
+enable_performance_tuning: true
```
Comment (VimukthiPerera) [inline dev/group_vars/apim.yml]: Please keep the default as false

File: README.md (hunk 1 above)
Comment (Savidude) [reply README.md]: Fixed https://github.com/wso2/ansible-apim/pull/45/commits/48626889a1dd269050e9be7a121221d2944181d1

File: dev/group_vars/apim.yml (hunk 2 above)
Comment (Savidude) [reply dev/group_vars/apim.yml]: Fixed https://github.com/wso2/ansible-apim/pull/45/commits/48626889a1dd269050e9be7a121221d2944181d1

**Pull request merged**